test/k8s-local-cluster-test/provision-cluster

```diff
@@ -9,6 +9,8 @@ OVERRIDE_PATH=0
 KIND_CONFIG_FILE=$SCRIPTPATH/kind-three-node-cluster.yaml
 use_psp=false
 
+# shellcheck disable=SC2034
+K8_1_29="kindest/node:v1.29.2@sha256:51a1434a5397193442f0be2a297b488b6c919ce8a3931be0ce822606ea5ca245"
 # shellcheck disable=SC2034
 K8_1_28="kindest/node:v1.28.0@sha256:b7a4cad12c197af3ba43202d3efe03246b3f0793f162afb40a33c923952d5b31"
```

Comment:
Does the patch version we use not matter? It seems like the latest release for kind has v1.29.2, v1.28.7, v1.27.11, v1.26.14, v1.25.16, v1.24.17, v1.23.17 listed as the latest images.